aips/objects.md

```diff
@@ -22,10 +22,10 @@ The object model allows Move on Aptos to represent a complex type as a set of re
 The object model improves type safety and data access through the use of a capability framework implemented by different accessors or references that define and distribute out various capabilities that allow for rich operations. These include:
 
 - `CreatorRef` that allows the creation of all other capabilities, allows adding resources to the object, and can only be accessed at the time the object is created
-- `OwnerRef` defines ownership of an object. Objects may have one or more `OwnerRef` and an `OwnerRef` can be aggregated, thus allow for fungability of objects.
-- `MintRef` allows for the creation of `OwnerRef`.
+- `OwnerRef` defines ownership of an object.
 - `DeleteRef` allows the holder to delete the object from storage.
```

Comment:
can we add more granularity here for removing from an object group (or is that party of `ExtendRef`?) vs deleting the object as a whole? What if someone puts a table into the object group?

Reply:
That would require ExtendRef. We could write more, but I think that might be better for giving a BaseObject example?

Reply:
I'm not sure what this means "removing from an object group" -- using an object group is optional -- I should clarify that above.